vmdb/lib/token_manager.rb

```diff
@@ -28,7 +22,13 @@ def self.class_initialize(name = DEFAULT_NS, options = {})
   end
 
   def self.global_token_store(namespace)
-    @global_token_store[namespace] ||= {}
+    @token_caches[namespace] ||= begin
+      memcache_server = VMDB::Config.new("vmdb").config[:session][:memcache_server] || "127.0.0.1:11221"
```

Comment:
Is this common to what is done in the initializer?  If so, should we move to a common module? cc @jrafanie